src/style.css

```diff
@@ -0,0 +1,118 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-items: center;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+}
+
+.card__price-text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-value {
+  color: #060b35;
+  font-weight: 700;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+  margin-top: 4px;
+  font-weight: 400;
+}
+
+.card__star {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  margin-top: 24px;
+  justify-content: space-between;
+}
+
+.card__star-reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+  display: flex;
+  align-items: center;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  text-transform: uppercase;
+  color: #fff;
+  font-size: 14px;
+  text-align: center;
+  text-decoration: none;
+  margin-top: 16px;
+  font-weight: 700;
+}
+
+.card__buy:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+  font-weight: 700;
+  transform: none;
+}
+
+.stars {
```

Comment:
better to divide by files, stars styles to stars.css, card to card.css and common styles keep in styles.css